src/styles/index.css

```diff
@@ -10,4 +10,4 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
-}
+} */
```

Comment:
This deleted CSS can just be deleted if you never plan to use it again :) The older version will be safely stored in git if you ever need it again.